include/EVT/dev/LCD.hpp

```diff
@@ -74,22 +77,86 @@ class LCD {
      */
     void displayMap(const uint8_t* bitMap);
```

Comment:
`displayMap` and `displayBitmap` have very similar names. Can you change them to make their usage a bit clearer? If you think their behavior is similar enough, you could also overload the function, so they have the same name.